.travis.yml

```diff
@@ -4,9 +4,6 @@ language: cpp
 os:
   - linux
   - osx
-compiler:
-  - clang
```

Comment:
platform_dependent is not building all tests, so there can be compile error catch by clang but not gcc. Hope this case can be covered.